### PR TITLE
Membership, Contact, and 404 Pages

### PR DESCRIPTION
**Pull request by tachorzy**
This PR includes three new pages: the Membership page, the Contact page, and the 404 page.
- Created 4 new components: `MembershipContact.tsx`, `MembershipInfoSection.tsx`, `PricingCard.tsx` and `EmailLink.tsx`.
- Added 404 page (`not-found.tsx`) 

**Next steps**
- Hamburger menu for mobile
- More responsiveness fixes, improve the `md` breakpoint page design (its the most lacking)
- Get headshots implemented. (Different image src per officer -- Waiting for headshots to be done)
- Update `README` for those who visit the repo.

### CODE REVIEW

File: app/contact/page.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image'
+import { satoshiMedium, satoshiBold } from '@/util/localFonts';
+import EmailLink from '@/components/EmailLink';
+
+export default function Contact() {
+    return (
+      // Deal with this code on the next sprint, too much to include in this sprint.
```
Comment (bnleft) [inline app/contact/page.tsx:7]: remove this comment

File: components/HomeButtonTray.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const trayOptions: string[][] = [
+    ["Join us!", "/"], //this will later link to the membership page once it's made
+    ["Socials", "/"], //this will later link to either a linktree or bento? Maybe even just the instagram. Depends on Marketing team.
+    ["Discord", "https://discord.gg/5Jhw67yQDH"],
+]
+
+const HomeButtonTray = () => {
+    return(
+        <div className = {satoshiBold.className}>
+          <div className="flex flex-row gap-x-8 md:max-2xl:gap-x-9 pt-3 md:max-2xl:pt-1 z-50">
```
Comment (bnleft) [inline components/HomeButtonTray.tsx:13]: nitpick: try to keep the spacing consistent for `=`

File: components/homepage/HomeButton.tsx [hunk 3]
```diff
@@ -7,9 +7,16 @@ interface ButtonProps {
 const HomeButton = (props: ButtonProps) => {
     return(
         <button className=" relative transition ease-in-out duration-500 delay-150 hover:scale-105 bg-white rounded-3xl w-20 h-[1.85rem] md:w-36 md:h-10 md:py-1.5 py-0.5 text-center z-50 overflow-hidden">
-            <Link href={props.link} target="_blank">
-                <h1 className="text-sm md:text-lg font-bold hover:text-cai-500 text-cai-400">{props.text}</h1>
-            </Link>
+            {/*If the link is within the website we'll use an anchor tag, otherwise we'll use the next/link component with a target attribute*/}
```
Comment (bnleft) [inline components/homepage/HomeButton.tsx:10]: This comment duplicates what the code shows below. 

Replace this comment so it explains why do we have to decide between `a` tag and `next/link` tag. Why does this nuance happen?

File: components/homepage/Logo.tsx [hunk 4]
```diff
@@ -1,12 +1,11 @@
 import Image from 'next/image'
 import { satoshiBold } from '@/util/localFonts';
 
-
 const Logo = () => {
     return (
         <div className={satoshiBold.className + " mt-28 md:mt-56 lg:mt-48 pr-3 z-40"}>
-            <Image src='/icons/CAI.webp' width={500} height={500} className="object-contain z-0 align-bottom select-none hidden md:block" alt="CAI Logo for desktop"/>
-            <Image src='/icons/CAI.webp' width={290} height={290} className="object-contain z-0 align-bottom select-none md:hidden block" alt="CAI Logo for mobile"/>
+            <Image src='/Icons/CAI.webp' width={500} height={500} className="object-contain z-0 align-bottom select-none hidden md:block" alt="CAI Logo for desktop"/>
```
Comment (bnleft) [inline components/homepage/Logo.tsx:7]: just curious, why the capitalization changes for `icons`

**Review by bnleft — CHANGES_REQUESTED** [review]
good work!

general nitpick comments
- keep the file naming consistent
  - ex: `public/icons` folder is a mix of snake_case, camelCase, and kebab-case
- choose either single or double quotes
- pick either 2 space or 4 space indentations

nitpick comments above don't need to be changed now but we will have to once we add a linter

File: components/homepage/Logo.tsx (hunk 4 above)
Comment (tachorzy) [reply components/homepage/Logo.tsx:7]: At some point the folder name got switched to `Icons` from `icons` but now it doesn't make much sense with every other folder being lowercase. I'll swap that over to be consistent.

**Review by bnleft — APPROVED** [review]

**Pull request merged**